docs/contributing/adding-images-to-the-wesite.md

```diff
@@ -0,0 +1,27 @@
+# Adding images to the design system website
+When adding or amending guidance on the design system website and you want to add an image, it's important to be considerate of the impact images have on the website. They can have implications for web performance and accessibility if used improperly.
+
+Before reading the following, also consult our [image guidance](https://design-system.service.gov.uk/styles/images/) for details on when it's appropriate to use images and how best to use them.
+
+## Does your content really need images?
+Ideally, we would avoid adding new images to our content.
```

Comment:
Maybe say "Generally" instead of ideally? 
I'm not sure we're opposed to images, more that it's uncommon for us to choose to use images. Do we need to explain why?

Reply:
I'm gonna leave it as changed to "generally" on the basis that this is internal guidance. I think that better represents that we avoid images unless it's the right thing, as you said.